src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/write.charMode..st

```diff
@@ -0,0 +1,4 @@
+writing
+write: anHex charMode: aBit
```

Comment:
I supposed that anHex is an integer? Then it is better to use anInteger as a name

Reply:
Yes, can be an integer also. But I put hexa because I'm using the hexa base (16r) on commands, so I thought in to keep the logic. But I can change if there is no sense.